### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by peppy**
I recall we were holding something back maybe. Let's see what CI says.

### CODE REVIEW

**Comment by bdach** [comment]
Tested this full-stack with https://github.com/ppy/osu/pull/30337 and it worked, but do you want to merge this now or keep open until those game-side bumps make a tag and then bump game packages here again? Probably less ceremony that way.

File: osu.Server.Spectator.Tests/osu.Server.Spectator.Tests.csproj [hunk 1]
```diff
@@ -9,14 +9,14 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
-        <PackageReference Include="Moq" Version="4.18.4" />
-        <PackageReference Include="xunit" Version="2.7.0" />
-        <PackageReference Include="xunit.runner.visualstudio" Version="2.5.7">
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
+        <PackageReference Include="Moq" Version="4.20.72" />
```
Comment (bdach) [inline osu.Server.Spectator.Tests/osu.Server.Spectator.Tests.csproj:13]: We can't touch this one ("sponsorlink" nonsense).

**Review by bdach — CHANGES_REQUESTED** [review]

File: osu.Server.Spectator.Tests/osu.Server.Spectator.Tests.csproj (hunk 1 above)
Comment (peppy) [reply osu.Server.Spectator.Tests/osu.Server.Spectator.Tests.csproj:13]: Oops, I thought I already reverted all of the `Moq` updates.

**Comment by peppy** [comment]
I've re-bumped `osu.Game` for https://github.com/ppy/osu-server-spectator/pull/245

**Pull request merged**